workers/tests/integrationtests/test_base_indicator.py

```diff
@@ -0,0 +1,38 @@
+import asyncio
+import unittest
+
+from ohsome_quality_analyst.geodatabase import client as db_client
+from ohsome_quality_analyst.indicators.ghs_pop_comparison_buildings.indicator import (
+    GhsPopComparisonBuildings,
+)
+
+
+class TestBaseIndicator(unittest.TestCase):
+    def test_geo_interface(self):
+        feature = asyncio.run(
+            db_client.get_feature_from_db(
+                dataset="regions", feature_id="3", fid_field="ogc_fid"
+            )
+        )
+        indicator = GhsPopComparisonBuildings(
+            feature=feature, layer_name="building_count"
+        )
+
+        feature = indicator.as_feature()
+        self.assertTrue(feature.is_valid)
```

Comment:
Wouldn't it be better to check the feature a little bit more explicitly (e.g. the content)?

Reply:
I added a check for some expected keys in the properties of the feature. 

I hope to define a schema for CLI and API output, where more detailed checks will be done.